tests/constants/dockercli/cmd.go

```diff
@@ -76,4 +76,13 @@ const (
 
 	// QueryContainer checks whether container exists or not
 	QueryContainer = docker + "ps -aq --filter name="
+
+	// StopAllContainers refers to stopping all the containers
```

Comment:
Nit: Just for consistency, let's remove "refers to" to keep the comments simple.